app/controllers/kyc/organisation-url/post.controller.js

```diff
@@ -22,6 +22,9 @@ function validateOrgUrl (organisationUrl) {
 
   const organisationUrlValidationResult = validateUrl(organisationUrl)
   if (!organisationUrlValidationResult.valid) {
+    logger.info('Blocked invalid organisation URL', {
```

Comment:
This functionality is not used anymore and will be removed soon so we don't need it here.

Reply:
We validate here https://github.com/alphagov/pay-selfservice/blob/38266a49c7f75bb70cd637328aab6912fc0bb7f5/app/controllers/request-to-go-live/organisation-address/post.controller.js#L79 as part of go-live flow